src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -47,6 +47,8 @@
     @Setter(AccessLevel.PRIVATE)
     private List<User> users = new ArrayList<>();
 
+    private Boolean isActive;
```

Comment:
Is there a reason this is `Boolean` with a capital-b instead of `boolean`?

Reply:
Originally it was lower case, but for some reason it couldn't compile and did not recognize `isActive` as a field of room that way. I'm not entirely sure why that is.

Reply:
Yeah I guess they need to be objects for some reason; I think the id is the same thing (Integer instead of int)